config/default.js

```diff
@@ -394,4 +395,19 @@ module.exports = {
   // in default-amo and/or default-disco depending on the target app for the
   // experiment.
   experiments: {},
+
+  // The list of GUIDs for add-ons that are Fenix compatible. This is based on
+  // the contents of the collection at
+  // https://addons.mozilla.org/firefox/collections/4757633/3204bb44a6ef44d39ee34917f28055/
+  fenixCompatibleGuids: [
```

Comment:
Did you run the disco app to make sure this new config does not break it?

Reply:
I did not. I thought that disco was virtually deprecated now, but I guess not quite yet?

Reply:
I just tested it. It seems fine.

Reply:
> I did not. I thought that disco was virtually deprecated now, but I guess not quite yet?

It is still in use.. (but not for a very long time)